dev_guide/templates.adoc

```diff
@@ -249,10 +249,9 @@ You can list the available default InstantApp templates with:
 $ oc get templates -n openshift
 ----
 
-ifdef::openshift-enterprise,openshift-origin[]
```

Comment:
This ifdef is needed for the Online/Dedicated builds (which don't have the Install & Config guide).